environment.yml

```diff
@@ -36,4 +36,4 @@ dependencies:
   - bioconda::subread=1.6.4
   - conda-forge::tqdm=4.45.0
   - pip:
-    - git+https://github.com/czbiohub/sencha.git@master#egg=sencha
+    - sencha==1.0.2
```

Comment:
https://pypi.org/project/sencha/ - released sencha so the translate timing out errors will stop